crates/accelerate/src/unitary_synthesis.rs

```diff
@@ -519,15 +519,15 @@ fn get_2q_decomposers_from_target(
     let qubits: SmallVec<[PhysicalQubit; 2]> = SmallVec::from_buf(*qubits);
     let reverse_qubits: SmallVec<[PhysicalQubit; 2]> = qubits.iter().rev().copied().collect();
```

Comment:
Do these even need to be a smallvec anymore? Like `qubits the input is already a slice, so we shouldn't need to do anything special here and maybe just do:

```suggestion
    let reverse_qubits: [PhysicalQubit; 2] = [qubits[1], qubits[0]];
```